accel-pppd/cli/tcp.c

```diff
@@ -52,7 +52,8 @@ static void disconnect(struct tcp_client_t *cln)
 {
 	struct buffer_t *b;
 
-	log_debug("cli: disconnect\n");
```

Comment:
it has debug level, if you don;t need debug messages just setup non-debug log-level.
verbose settings usually applies only to info-level messages

Reply:
more easy is to add :

if (conf_verbose)
log_debug("cli: disconnect\n");

no need to remove debug message more easy to add in verbose mode.